### ISSUE TITLE
Feature: Internationalize CLI output

### ISSUE THREAD

**Issue by gronke** [iocage/libiocage#49]
Introduced with #37 there will be signaling between the library and the cli, which is responsible for printing event to the user. All events are listed in `libiocage/lib/events.py`. Another list of event-like items is `libiocage/lib/errors.py`. Both are inviting for internationalization!

To generate similar output to previous iocage versions, we should begin with adding translations for English language.

We may wait with translation iocage to further languages until we come close to a stable release, so that we don't have update translations among changing interfaces.
